cmd/explaintest/run-tests.sh

```diff
@@ -97,10 +94,6 @@ while getopts "t:s:r:b:v:c:i:h" opt; do
                     ;;
             esac
             ;;
-        v)
```

Comment:
should we also remove this flag in other "run-all-tests.sh"? for example, mysql-test, randgen-test